hardhat.config.ts

```diff
@@ -40,14 +49,22 @@ const config: HardhatUserConfig = {
     wallets: {
       OpenZeppelin: {
         accountName: "OpenZeppelin",
-        modulePath: "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount",
-        accountPath: "~/.starknet_accounts"
-      }
-    }
+        modulePath:
+          "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount",
+        accountPath: "~/.starknet_accounts",
+      },
+    },
   },
   networks: {
     devnet: {
```

Comment:
if it's possible, we could rename this network to `l2-testnet` to have a consistent naming accross chains

Reply:
agreed